spec/atom-reporter.coffee

```diff
@@ -76,6 +107,10 @@ class AtomReporter extends View
     @on 'click', '.stack-trace', ->
       $(this).toggleClass('expanded')
 
+    @on 'click', '.stack-trace a', ->
+      if href = $(this)[0].dataset.href
```

Comment:
using `dataset` because the `href` attribute was screwing with the path, making it a URI, which then prevented parsing the line/char numbers in `AtomApplication::locationForPathToOpen`